infra/db.bicep

```diff
@@ -41,25 +61,30 @@ resource postgresConfig 'Microsoft.DBforPostgreSQL/flexibleServers/configuration
   }
 }
 
-resource sqlDeploymentScript 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
-  name: '${name}-deployment-script'
+resource sqlDeploymentScriptSetup 'Microsoft.Resources/deploymentScripts@2020-10-01' = {
+  name: '${name}-deployment-script-setup'
   dependsOn: [
     postgresConfig
   ]
   location: location
   kind: 'AzureCLI'
   properties: {
-    azCliVersion: '2.37.0'
+    azCliVersion: '2.9.1'
```

Comment:
This looks like a large downgrade in the version used, is that going to be a problem? I'd prefer we don't downgrade the CLI version